indico/util/mathjax.py

```diff
@@ -1,5 +1,5 @@
 # This file is part of Indico.
-# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
```

Comment:
I'd amend the commit where you add the file instead of adding a new commit for it